### PR TITLE
Add improvements.

### PR DESCRIPTION
**Pull request by dewniMW**

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ nav:
     - 'Contribute': get-started/community.md
     - 'Feature Tour': get-started/quick-start-guide.md
   - 'Quick Starts':
-      - 'Overview': quick-starts/overview.md
+      #- 'Overview': quick-starts/overview.md
```
Comment (gomathyk) [inline en/mkdocs.yml:59]: Is this intentional?

File: en/mkdocs.yml [hunk 2]
```diff
@@ -115,7 +115,7 @@ nav:
         - 'Twitter': quick-starts/twitter-as-federated-authenticator-sample.md
         - 'Microsoft Azure': quick-starts/ms-azure-as-federated-authenticator-sample.md
   - 'Guides':
-      - 'Get Started': guides/before-you-start.md
+      #- 'Get Started': guides/before-you-start.md
```
Comment (gomathyk) [inline en/mkdocs.yml:118]: Is this intentional?

**Review by gomathyk — APPROVED** [review]

File: en/mkdocs.yml (hunk 2 above)
Comment (dewniMW) [reply en/mkdocs.yml:118]: yes as we don't have an overview page for this section

File: en/mkdocs.yml (hunk 1 above)
Comment (dewniMW) [reply en/mkdocs.yml:59]: yes as we don't have an overview page for this section

**Pull request merged**